### DISCUSSION TITLE
Not able to bind GSPRO to connector getting API error

### DISCUSSION THREAD

**Discussion by Amwldinho** [springbok/MLM2PRO-GSPro-Connector#162] [General]
![Unbenannt2](https://github.com/user-attachments/assets/58f512a0-d90d-428f-a630-d6aa40003dfe)
Does anybody know how to fix the issue in the attached screenshot?! I am not able to bind GSPRO and getting this error.
Is it because I am on Windows 10 instead 11 or is it possibly related to framenetwork of Windows?!

restarting GSPro, laptop or API window did not solve it.

Any help to solve is appriciated.

Thanksl.

**Comment by springbok** [comment]
Can you post a picture of the GSPro API window, also upload the log file in appdata\logs directory Sent from my iPhoneOn 17 Dec 2024, at 11:28 pm, Amwldinho ***@***.***> wrote:﻿
Unbenannt2.PNG (view on web)
Does anybody know how to fix the issue in the attached screenshot?! I am not able to bind GSPRO and getting this error.
Is it because I am on Windows 10 instead 11 or is it possibly related to framenetwork of Windows?!
restarting GSPro, laptop or API window did not solve it.
Any help to solve is appriciated.
Thanksl.

**Reply by Amwldinho** [reply]
here we go. But in the meantime I am using the garmin R10 which works without the connector with GSPRO but I need the tool for webcam puttin or is there another option, to get the webcam putting working with GSpro?!
![Unbenannt  11](https://github.com/user-attachments/assets/35d2843c-e491-4326-9b7f-335619bf5475)
[mlm2pro-gspro-connect.log](https://github.com/user-attachments/files/18180208/mlm2pro-gspro-connect.log)

**Comment by springbok** [comment]
You are running GSPro using the R10 GSPro license by the looks of it, the connecter uses the OpenAPI license. If you want to use the connector you will need to switch license types.

**Comment by gmsiegel** [comment]
@Amwldinho - I believe you may just need to reset your GSPro connection. 

Try this:
Launch **GSPro**. Go to **Settings\System** and click the **Reset GSPro Connect** button
